package.json

```diff
@@ -4,15 +4,32 @@
   "private": true,
   "description": "Typings for vinyl",
   "scripts": {
-    "build": "typings bundle -o test/bundle.d.ts",
-    "test": "tsc -p test",
-    "lint": "tslint -c tslint.json index.d.ts"
+    "build": "echo building... && typings bundle -o out/index.d.ts",
+    "lint": "echo linting... && tslint \"**/*.ts\" -e \"source/**\" -e \"source-test/**\" -e \"out/**\" -e \"node_modules/**\" -e \"typings/**\"",
+    "browser-test": "echo running browser tests... && echo no browser test",
+    "source-test": "echo running source tests... && echo no source test",
+    "test": "echo running server tests... && cd test && ts-node ../node_modules/blue-tape/bin/blue-tape \"**/*.ts\" | tap-spec",
+    "watch": "onchange -w \"**/*.ts\" -i -e \"out/**\" -- npm -s run build+test",
+    "publish": "npm -s run lint+build+test && echo please publish to typings/registry",
+    "all-tests": "npm test && npm run source-test",
+    "build+test": "npm run build && npm run all-tests",
+    "lint+build+test": "npm run lint && npm run build+test",
```

Comment:
Just keep the old test scripts and keep the tests. Removing the tests is a no-go